yarn.lock

```diff
@@ -11751,7 +11751,7 @@ safe-regex@^1.1.0:
   dependencies:
     ret "~0.1.10"
 
-"safer-buffer@>= 2.1.2 < 3", safer-buffer@^2.1.0, safer-buffer@~2.1.0:
+"safer-buffer@>= 2.1.2 < 3", safer-buffer@^2.0.2, safer-buffer@^2.1.0, safer-buffer@~2.1.0:
```

Comment:
Should this be changing if the `package.json` hasn't changed?

Reply:
I just cloned it for the first time and ran `yarn`. Must not have been committed before @ashleynolan

Reply:
Or a package may not have been pinned??